sdks/python/apache_beam/io/mongodbio.py

```diff
@@ -204,7 +205,7 @@ def display_data(self):
     res['collection'] = self.coll
     res['filter'] = self.filter
     res['project'] = self.projection
```

Comment:
Do you need to convert the projection as well?

Reply:
yes, this is included in https://github.com/apache/beam/pull/9656 and I'm gonna cherry-pick separately

Reply:
or actually I think we can merge two in this.